mods/cnc/rules/defaults.yaml

```diff
@@ -122,9 +122,12 @@
 		CloakDelay: 90
 		CloakSound: trans1.aud
 		UncloakSound: trans1.aud
-		RequiresCondition: cloak
+		RequiresCondition: cloak && !uncloak
 	ExternalConditions@CLOAK:
 		Conditions: cloak
+	GrantConditionOnDamageState@UNCLOAK:
+		Condition: uncloak
```

Comment:
Then on the other hand, it could be rename it to more explit in-critical-health, possibly to share this condition name. In TS, units slow down when in critical health. Which change do you prefer?